### DISCUSSION TITLE
클린 아키텍처 개요와 기본 개념

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#226] [General]
1일: 클린 아키텍처 개요와 기본 개념

1: 클린 아키텍처 소개 및 기본 원칙
2: 엔티티(Entity) 개념과 역할
3: 레포지토리(Repository) 패턴과 중요성
4: 데이터 변환(Transform) 및 매핑
5: 레포지토리 인터페이스(Repository Interface) 설계

이거 공부하고 11/10 올리시면 됩니다~!

**Comment by Eunho0922** [comment]
넵 !
